README.md

```diff
@@ -0,0 +1,31 @@
+# time-rs book
+
+[![Build](https://github.com/time-rs/book/actions/workflows/build.yaml/badge.svg)](https://github.com/time-rs/book/actions/workflows/build.yaml)
+
+## time documentation
+
+This repository contains the [mdbook](https://github.com/rust-lang/mdBook) source for the [time-rs](https://github.com/time-rs/time) crate [book](https://time-rs.github.io/book/).
+
+## Contributing
+
+Contributions are always welcome! If you have an idea, it's best to float it by me before working on
+it to ensure no effort is wasted. If there's already an open issue for it, knock yourself out.
+Internal documentation can be viewed [here](https://time-rs.github.io/internal-api/time).
+
+If you have any questions, feel free to use [Discussions]. Don't hesitate to ask questions — that's
+what I'm here for!
+
+[Discussions]: https://github.com/time-rs/time/discussions
+
+## License
+
+This project is licensed under either of
+
+- [Apache License, Version 2.0](https://github.com/time-rs/time/blob/main/LICENSE-Apache)
+- [MIT license](https://github.com/time-rs/time/blob/main/LICENSE-MIT)
```

Comment:
These should link to the correct repository.

Reply:
Fixed the links to point to this repos's LICENSE files.